scripts/schema_preprocess.py

```diff
@@ -0,0 +1,704 @@
+# -*- mode:python; coding:utf-8 -*-
```

Comment:
I'm not sure I understand the changes that have resulted in the manual hacking to the degree established here. However, I've been out of this for too long understand why.

Reply:
Fixed.

Improved module comments delivered.

Reply:
Okay I've put in #1561 as an umbrella for a forward looking strategy.